.zuul/playbooks/containerd-build/run.yaml

```diff
@@ -0,0 +1,19 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: Build containerd
+      shell:
+        cmd: |
+          set -xe
+          apt-get update
+          apt-get install -y btrfs-tools libseccomp-dev git pkg-config
+          apt-get install -y golang-1.10-go
```

Comment:
You can use OpenLab ansible role `config-golang` to enable specific golang.

Reply:
done